content/administration/maintain/odoo_online.rst

```diff
@@ -2,21 +2,26 @@
 Odoo Online database management
 ===============================
 
+Database management
+===================
+
 To manage a database, sign in to https://www.odoo.com and access the `database management page
-<https://www.odoo.com/my/databases>`_ by clicking on the user icon, then on *My Databases*.
+<https://www.odoo.com/my/databases>`_ by clicking on the user icon, then on
+:guilabel:`My Databases`.
 
 .. image:: odoo_online/my-databases.png
    :align: center
    :alt: Clicking on the user icon opens a drop-down menu. "My databases" button is highlighted.
 
 .. note::
-   Make sure you are connected as the administrator of the database you want to manage.
+   Make sure to connect as the administrator of the database that changes will be made on.
 
 .. image:: odoo_online/dropdown-menu.png
    :align: right
    :alt: Clicking on the gear icon opens the drop-down menu.
 
-Open the drop-down menu next to the database you want to manage by clicking on the gear icon.
+Open the drop-down menu next to the database that changes will be made on by clicking on the gear
```

Comment:
Should the gear icon be a guilabel?

Reply:
I was told by Jonathan that it isn't a guilabel.